lib/presentation/widgets/series_element.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:marvelapp_flutter/data/model/response_models/series.dart';
+
+class SeriesElement extends StatelessWidget {
+  final Series? series;
+
+  const SeriesElement({Key? key, this.series}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: SizedBox(
+        width: 120,
+        height: 170,
+        child: Column(
+          children: [
+            FadeInImage.assetNetwork(
+                placeholder: "assets/graphics/placeholder-150x150.png",
+                image:
+                    "${series?.thumbnailPath}/portrait_medium.${series?.thumbnailExtension}",
+                width: 120.0,
+                height: 170.0,
+                fit: BoxFit.fitWidth),
+            Text(series?.title ?? "title missing",
```

Comment:
If something non-critical is missing - just do not show it, but if it is critical for loading the page or widget - show the error state. This case looks like a non-critical.

Reply:
done